### ISSUE TITLE
calc not parsed correctly

### ISSUE THREAD

**Issue by david-truong** [phax/ph-css#13]
the follow expression fails for me: calc((100% - (3 * 30px)) / 4)

**Comment by phax** [comment]
Thanks, fixed in 5.0.0-SNAPSHOT

**Comment by david-truong** [comment]
any chance this can be backported to 4.x. I'm still on a Java 7 project.

**Comment by phax** [comment]
I will collect some issues and once 5.0.0 is ready I will create a backport version 4.1.0 that contains all the fixes - okay?
Would be great if you could open a separate issue for this :) Thanks!

**Comment by phax** [comment]
(Also because some \@page rules were enhanced)

**Comment by david-truong** [comment]
awesome thanks!
